DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/staticIndexesOverview/metadata.json

```diff
@@ -0,0 +1,237 @@
+        },
+        //TODO
+        //{
+        //    "Title": "Creating and Deploying Indexes",
+        //    "Type" : "link",
+        //    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/creating-and-deploying#indexes--creating-and-deploying-indexes"
+        //}, 
+        {
+            "Title": "Querying a Specified Index",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/querying/basics#example-iv---querying-a-specified-index"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define the index class",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Create an index class that inherits from a common abstract index class.",
+                "",
+                "* Use:<br>",
+                "    * `AbstractIndexCreationTask` - for a Map index<br>",
+                "    * `AbstractMultiMapIndexCreationTask` - for a Multi-Map index<br>",
+                "    * `AbstractJavaScriptIndexCreationTask` - for a Javascript index",
+                "",
+                "* Note:<br>",
+                "     The character '_' in the index name will be changed to _'/'_ when the index is deployed.",
+                ""
+            ],
+            "Assets": [
+                {
+                    "Title": "Indexes Overview",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/studio/database/indexes/indexes-overview"
+                },
+                {
+                    "Title": "What Indexes Are",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/what-are-indexes#what-indexes-are"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/creating-and-deploying#static-indexes"
+                },
+                {
+                    "Title": "Defining simple indexes",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch12/Ch12.md#defining-simple-indexes-via-client-code"
+                }
+            ]
+        },
+        {
+            "Title": "The index Result class",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* The Result class defines the _index-fields_ - the fields that will be indexed.<br>",
+                "  Later on, a query can be made on these index-fields.",
+                "",
+                "* The _content_ for each index-field is configured in the index constructor (see next step).",                
+                "",
+                "* In the above index, field 'lastName' is indexed and will be queryable."
```

Comment:
LastName?

Reply:
This text is supposed to explain the 'result' class (which contains 'lastName')
Wouldn't it be confusing to have 'LastName' ?

Reply:
no, index doesn't index result classes. Index works on documents. And document has field LastName. 

So, it goes like this: field LastName from JSON document is indexed, then you can query LastName field and assign to field lastName in result class.